extensions/amp-ad-exit/0.1/config.js

```diff
@@ -138,3 +148,29 @@ function assertTarget(name, target, config) {
     }
   }
 }
+
+/**
+ * Checks whether a vendor is valid (i.e. listed in vendors.js and has
+ * transport/iframe defined.
+ * @param {string} vendor The vendor name that should be listed in vendors.js
+ */
+function assertVendor(vendor) {
+  user().assert(ANALYTICS_CONFIG &&
+      ANALYTICS_CONFIG[vendor] &&
+      ANALYTICS_CONFIG[vendor]['transport'] !== undefined &&
+      ANALYTICS_CONFIG[vendor]['transport']['iframe'] !== undefined,
+      'Unknown vendor: ' + vendor);
+}
+
+/**
+ * Ensures that a given origin matches that of an existing vendor's
+ * transport/iframe URL
+ * @param {string} origin The origin to verify
+ * @param {string} vendor The vendor whose origin to check against
+ */
+export function assertOriginMatchesVendor(origin, vendor) {
```

Comment:
given this is only used in amp-ad-exit, let's move to that file and make private

Reply:
Done